modules/volume/rendering/basicvolumeraycaster.cpp

```diff
@@ -32,44 +32,44 @@
 #include <openspace/util/powerscaledcoordinate.h>
 #include <openspace/util/updatestructures.h>
 #include <openspace/rendering/renderable.h>
-#include <modules/kameleonvolume/rendering/renderablekameleonvolume.h>
-
 
 namespace {
-    const char* GlslRaycastPath = "${MODULES}/kameleonvolume/shaders/raycast.glsl";
-    const char* GlslHelperPath = "${MODULES}/kameleonvolume/shaders/helper.glsl";
-    const char* GlslBoundsVsPath = "${MODULES}/kameleonvolume/shaders/boundsvs.glsl";
-    const char* GlslBoundsFsPath = "${MODULES}/kameleonvolume/shaders/boundsfs.glsl";
+    const char* GlslRaycastPath = "${MODULES}/volume/shaders/raycast.glsl";
```

Comment:
You can use `${MODULE_VOLUME}` .. that file system token is generated automatically